public/data/Dongwu/feedData.json

```
@@ -0,0 +1,23 @@
+[
+  {
+    "id": 1,
+    "feedProfileImg": "/images/Dongwu/Main/freeImage.png",
+    "feedImg": "/images/Dongwu/Main/freeImage.png",
+    "feedId": "steam_udon1",
+    "feedText": "가나다라마바사"
```

```suggestion
    "profile_img": "/images/Dongwu/Main/freeImage.png",
    "feed_img": "/images/Dongwu/Main/freeImage.png",
    "feed_id": "steam_udon1",
    "feed_content": "가나다라마바사"
```
mock data는 보통 서버에서 넘어오는 데이터들을 모방하게 되는데, 보통 snake_case로 오는거 같아요.